### PR TITLE
Scripts tests

### PR DESCRIPTION
**Pull request by johnpyp**

### CODE REVIEW

File: scripts/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "benchmark:index": "node -r dotenv/config ./src/benchmark-ingest.js",
     "lint": "prettier-eslint --write '**/*.js'",
     "check-lint": "eslint --ext .js ./",
-    "test": "jest --forceExit"
+    "test": "DATA_PATH=test-data jest --forceExit"
```
Comment (alg1142) [inline scripts/package.json]: doesnt have to be here since its already in the env from the drone config

File: scripts/package.json [hunk 2]
```diff
@@ -52,7 +52,9 @@
     "@elastic/elasticsearch": "7.1.0",
     "bluebird": "3.5.5",
     "etl": "0.6.4",
+    "fs-extra": "^8.1.0",
     "luxon": "1.16.0",
+    "nock": "^10.0.6",
```
Comment (alg1142) [inline scripts/package.json]: no ^ versions

File: scripts/package.json (hunk 1 above)
Comment (johnpyp) [reply scripts/package.json]: I want to keep in case someone manually uses `yarn test`. It is repetitive, but it is a safeguard for the real data.

Comment (alg1142) [reply scripts/package.json]: sure, that works

**Pull request merged**